src/main/java/views/MainMenuPane.java

```diff
@@ -0,0 +1,80 @@
+package views;
+
+import javafx.application.Platform;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import controllers.SceneManager;
+
+/**
+ * Pane for displaying the main menu.
+ */
+public class MainMenuPane extends GamePane {
+
+    private final VBox container = new VBox();
+
+    private final Label title = new Label("Fantasy Soccer");
+
+    private final Button leagueSimulatorButton = new Button("League simulator");
+
+    private final Button teamEditorButton = new Button("Team editor");
+
+    private final Button playerEditorButton = new Button("Player editor");
+
+    private final Button leaderboardButton = new Button("Leaderboard");
+
+    private final Button quitButton = new Button("Quit");
+
+    public MainMenuPane() {
+        connectComponents();
+        styleComponents();
+        setCallbacks();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    void connectComponents() {
+        // TODO
+        container.getChildren().addAll(title, leagueSimulatorButton, leaderboardButton, teamEditorButton, playerEditorButton, quitButton);
+        container.setAlignment(Pos.CENTER);
+        this.setCenter(container);
+
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    void styleComponents() {
+
+        String bigButton = "    -fx-font-size: 15;\n" +
+                "    -fx-font-family: sans-serif;\n" +
```

Comment:
I'd indent these a bit more such that the quotes all start at the same column! Why is there some whitespace following the quotes? Not sure if it matters but could be removed if it doesn't.